src/euphorie/client/model.py

```diff
@@ -801,20 +802,42 @@ def is_archived(self):
             return False
         return archived <= localized_now()
 
+    @property
+    def last_locking_event(self):
+        """Return the last event relative to locking"""
+        query = (
+            Session.query(SessionEvent)
+            .filter(
+                SessionEvent.action.in_(("lock_set", "lock_reset", "lock_unset")),
```

Comment:
Use `LOCKING_ACTIONS` from config

```suggestion
                SessionEvent.action.in_(LOCKING_ACTIONS),
```